resources/definitions/ultimaker_method_base.def.json

```diff
@@ -354,11 +354,12 @@
         "print_sequence": { "enabled": false },
         "raft_base_line_spacing": { "value": "2*raft_base_line_width" },
         "raft_base_line_width": { "value": 1.4 },
-        "raft_base_speed": { "value": 5 },
+        "raft_base_speed": { "value": 10 },
         "raft_base_thickness": { "value": 0.8 },
         "raft_interface_extruder_nr": { "value": "raft_surface_extruder_nr" },
         "raft_interface_layers": { "value": 2 },
-        "raft_interface_line_width": { "value": 1.2 },
+        "raft_interface_line_width": { "value": 0.7 },
```

Comment:
Reduced line width of the middle raft layer from 1.2 to 0.4
This is what is mentioned in PR. but here it's changed to 0.7?

Reply:
@saumyaj3 The latest tests from @alanbjorklund showed that the line width could be increased from 0.4mm to 0.7mm with the same bottom quality. This change from 0.4mm to 0.7mm improve the productivity while keeping the quality intact. I changed the comments in the PR to reflect the new setting.